packages/workers-shared/package.json

```diff
@@ -26,7 +26,7 @@
 		"bundle:asset-worker": "esbuild asset-server-worker/src/index.ts --format=esm --bundle --outfile=dist/asset-server-worker.mjs --sourcemap=external --external:cloudflare:*",
 		"bundle:asset-worker:prod": "pnpm run bundle:asset-worker --minify",
 		"bundle:router-worker": "esbuild router-worker/src/index.ts --format=esm --bundle --outfile=dist/router-worker.mjs --sourcemap=external",
-		"bundle:router-worker:prod": "pnpm run bundle:router-worker --minify",
+		"bundle:router-worker:prod": "pnpm run bundle:router-worker --minify && node -r esbuild-register scripts/copy-config-file.ts",
```

Comment:
I think we need to be copying the asset worker toml? Since the router worker is not currently being imported

Reply:
oh darn it...wrong script 😅

Reply:
fixed ✅